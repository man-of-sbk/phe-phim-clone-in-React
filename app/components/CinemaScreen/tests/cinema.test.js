import React from 'react';
import { render } from '@testing-library/react';

import Cinema from '../index';

describe('<Cinema /> Component', () => {
  it('Should correct snapshot', () => {
    const { container } = render(<Cinema />);
    const cinema = container.firstChild;
    expect(cinema).toMatchSnapshot();
  });
});
