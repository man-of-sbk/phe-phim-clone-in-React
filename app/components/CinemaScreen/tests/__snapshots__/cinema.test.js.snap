// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cinema /> Component Should correct snapshot 1`] = `
<div
  class="Wrapper-l33txi-0 inoUTl"
>
  <div
    class="screen"
  />
  <div
    class="screen-brightness"
  >
    <div
      class="outer-mask"
    >
      <div
        class="inner-mask"
      >
        <div
          class="screen-light"
        />
      </div>
    </div>
    <div
      class="content"
    >
      Screen
    </div>
  </div>
</div>
`;